dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -312,6 +314,33 @@ export const Card = ({
 		format.design === ArticleDesign.Editorial ||
 		format.design === ArticleDesign.Letter;
 
+	const decideAge = () => {
+		if (!webPublicationDate) return undefined;
+
+		const publishedWithinTwelveHours =
+			isWithinTwelveHours(webPublicationDate);
+
+		const shouldShowAge =
+			isTagPage ||
+			!!onwardsSource ||
+			(showAge && publishedWithinTwelveHours);
+
+		if (!shouldShowAge) return undefined;
+
+		return (
+			<CardAge
+				format={format}
+				containerPalette={containerPalette}
+				webPublicationDate={webPublicationDate}
+				showClock={showClock}
+				isDynamo={isDynamo}
+				isOnwardContent={isOnwardContent}
+				absoluteServerTimes={absoluteServerTimes}
+				isTagPage={isTagPage}
+				isWithinTwelveHours={publishedWithinTwelveHours}
+			/>
+		);
+	};
```

Comment:
If we chose to extract this function outside of this component and pass all the required parameters to it, and we could test it.

Not a blocker for now as this is an actual fix, but it could be valuable?

Reply:
Good suggestion, i'll follow this up in an further PR so we can roll this fix out first